ui/panels-plugin/src/plugins/gauge-chart/GaugeChartOptionsEditorSettings.tsx

```diff
@@ -53,6 +55,26 @@ export function GaugeChartOptionsEditorSettings(props: GaugeChartOptionsEditorPr
         <OptionsEditorGroup title="Misc">
           <UnitSelector value={unit} onChange={handleUnitChange} />
           <CalculationSelector value={value.calculation ?? DEFAULT_CALCULATION} onChange={handleCalculationChange} />
+          <OptionsEditorControl
+            label="Max"
+            control={
+              <TextField
+                type="number"
+                value={value.max ?? ''}
+                onChange={(e) => {
+                  // ensure empty value resets to undef to allow chart to calculate max since
+                  // max only needs to be set explicitly for units other than Percent and PercentDecimal
+                  const newValue = e.target.value ? Number(e.target.value) : undefined;
+                  onChange(
+                    produce(value, (draft: GaugeChartOptions) => {
+                      draft.max = newValue;
+                    })
+                  );
+                }}
+                placeholder="Default"
+              />
```

Comment:
A few UX questions:
- Would it make sense to show the defaults in the text box for the cases where this isn't required?
- Should we have some UI to communicate when this is required (the not-percent case)?

These are both incremental improvements that could be done in a follow-up after consultation with design.

Reply:
Yeah good idea! I updated the max placeholder so it shows the default value depending on unit.kind, see example here:

https://user-images.githubusercontent.com/9369625/210578538-e9193457-b17b-4591-a3b9-489742ea4c18.mp4